packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -94,6 +96,11 @@ const TextGridItem = styled.div`
   }
 `;
 
+const InlineMediaIndicator = styled(MediaIndicator)`
```

Comment:
Is this working for you? Because the `InlineMediaIndicator` is not rendered in my local and neither the styles associated with it. 

To make it work I had to pass a `className` prop to the `MediaIndiciator` as it explains [here](https://www.styled-components.com/docs/basics#styling-any-components):

```
const MediaIndicator = ({ className, datetime, duration, offscreenText }) => (
  <MediaIndicatorWrapper className={className}>
```

Reply:
should be fixed now. I had many issues when I first started working on this PR with changes not being picked up. I used that example of adding `className` which worked but removed it later, thinking it would work, clearly it didn't 🙈